web/src/portico/google-analytics.ts

```diff
@@ -0,0 +1,15 @@
+import {gtag, install} from "ga-gtag";
+import type {ConfigParams} from "ga-gtag";
+
+import {page_params} from "../base_page_params";
+
+export let config: (info: ConfigParams) => void;
+
+if (page_params.google_analytics_id !== undefined) {
+    install(page_params.google_analytics_id);
+    config = (info) => {
+        gtag("config", page_params.google_analytics_id!, info);
+    };
+} else {
+    config = () => undefined;
```

Comment:
`gtag` returns `void` so replacing `{}` with `undefined` is safe here, also we anyways do not use the returned value of `config` function anywhere.

Reply:
The point of the ESLint `no-empty-function` rule is to suggest writing a comment that explains why the body is empty, not rewriting with a different syntax to evade the rule.

Reply:
Got it, added a comment in the function body.